docker/images.go

```diff
@@ -1,36 +1,101 @@
 package docker
 
 import (
+	"context"
 	"fmt"
+	"io"
 	"log"
+	"os"
 	"path/filepath"
+	"strings"
 	"time"
 
 	"github.com/briandowns/spinner"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/pkg/jsonmessage"
 	"github.com/spf13/viper"
 	"golang.org/x/sync/errgroup"
 	"liferay.com/liferay/cli/ansicolor"
 	"liferay.com/liferay/cli/constants"
+	lstrings "liferay.com/liferay/cli/strings"
 )
 
 func BuildImages(verbose bool) {
 	var s *spinner.Spinner
 
+	image := viper.GetString(constants.Const.DockerLocaldevServerImage)
+
+	if viper.GetBool(constants.Const.DockerLocaldevServerPullimage) {
```

Comment:
I'm assuming that eventually we want to provide some type of migration or on-ramp to this setting.  When I'm trying this out and I will have to manually set both image and pullImage configs to their new values.  Existing users will still be pointing to localdev local image.